src/views/News.vue

```diff
@@ -1,20 +1,32 @@
 <template>
-  <div :class="main">
+  <div>
     <el-header>
       <el-input
-        v-model="input"
+        v-model="inputText"
         placeholder="请输入内容"
         class="search"
       ></el-input>
       <el-button icon="el-icon-search" circle></el-button>
     </el-header>
-    <el-main>
-      <more-article-item
-        v-for="info in infos"
-        :key="info.id"
-        :info="info"
-      ></more-article-item>
-    </el-main>
+    <el-container class="main">
+      <el-aside>
+        <a href="#" @click="handleCheckOut(1)">筛选新闻条件一</a>
+        <a href="#" @click="handleCheckOut(2)">筛选新闻条件二</a>
+        <a href="#" @click="handleCheckOut(3)">筛选新闻条件三</a>
+        <a href="#" @click="handleCheckOut(4)">筛选新闻条件四</a>
+        <a href="#" @click="handleCheckOut(5)">筛选新闻条件五</a>
+      </el-aside>
+      <el-main>
+        <transition-group appear="True" name="el-fade-in">
```

Comment:
```suggestion
        <transition-group appear name="el-fade-in">
```
`appear`不用赋值。就算要赋值，因为它是个`boolean`，也应该写成`:appear="true"`。

Reply:
已处理